osvr_tracked_device.h

```diff
@@ -488,7 +455,16 @@ const char* OSVRTrackedDevice::GetSerialNumber()
 
 float OSVRTrackedDevice::GetIPD()
 {
-    return static_cast<float>(m_DisplayConfiguration->getIPDMeters());
+    OSVR_Pose3 leftEye, rightEye;
+    if(m_DisplayConfig.getViewer(0).getEye(0).getPose(leftEye) != true) {
+        logger_->Log("OSVRTrackedDevice::GetHeadFromEyePose(): Unable to get left eye pose!\n");
+    }
+    if(m_DisplayConfig.getViewer(0).getEye(1).getPose(rightEye) != true) {
+        logger_->Log("OSVRTrackedDevice::GetHeadFromEyePose(): Unable to get right eye pose!\n");
+    }
+    Eigen::Map<Eigen::Vector3d> lT = osvr::util::vecMap(leftEye.translation);
+    Eigen::Map<Eigen::Vector3d> rT = osvr::util::vecMap(rightEye.translation);
+    return 2*std::sqrt(std::abs(rT.dot(lT)));
```

Comment:
I have no idea - should be able to just do `(osvr::util::vecMap(leftEye.translation) - osvr::util::vecMap(rightEye.translation)).norm()` to get the distance between the two eyes.

See <http://eigen.tuxfamily.org/dox/classEigen_1_1MatrixBase.html#a0be1b433c65ce9d92c81a4718daf54e5>